src/hotspot/cpu/aarch64/macroAssembler_aarch64_log.cpp

```diff
@@ -297,7 +297,8 @@ void MacroAssembler::fast_log(FloatRegister vtmp0, FloatRegister vtmp1,
   bind(MAIN);
     fmovs(tmp3, vtmp5);                        // int intB0 = AS_INT_BITS(B);
     mov(tmp5, 0x3FE0);
-    mov(rscratch1, 0xffffe00000000000);
+    uint64_t mask = 0xffffe00000000000UL;
```

Comment:
I think this should be using ULL to support LLP64.

Reply:
Integral literal takes the first type that fits so for this 64-bit all U, UL and ULL would produce the same type.

Reply:
Thanks, I wasn't sure what would happen if you used the "wrong" suffix. I think it still makes sense to use the correct prefix for the intended target type.

Reply:
I changed it to ULL. Thanks.

Reply:
We have `CONST64` and `UCONST64` for these situations (in globalDefinitions.hpp).

Reply:
Forgot about those. thanks.